entrypoint.R

```diff
@@ -7,6 +7,9 @@ use_archive <- Sys.getenv("CRANIUM_USE_ARCHIVE")
 use_hardlinks <- Sys.getenv("CRANIUM_USE_HARDLINKS")
 latest_only <- Sys.getenv("CRANIUM_LATEST_ONLY")
 fields <- Sys.getenv("CRANIUM_REPO_FIELDS")
+host_addr <- Sys.getenv("CRANIUM_HOST_ADDRESS", "0.0.0.0")
```

Comment:
I don't think we want this -- Docker won't allow the host to be anything other than `0.0.0.0` inside the container.

Reply:
Right, this is mainly for outside the container or in unusual network situations.